### ISSUE TITLE
printing multiple pages doesn't put the header on first page but on all following pages

### ISSUE THREAD

**Issue by josefstr** [getgrav/grav-theme-antimatter#69]
If you print simple text, which is longer then 1 page, you get no header on the first page but on all other pages 2 to n you get a header which also cuts some of the text of.

**Comment by flaviocopes** [comment]
You mean print on paper?

**Comment by flaviocopes** [comment]
Yep the theme has no print CSS, if you want to look at providing one, a PR is well accepted.
